src/store/reducers/article.js

```diff
@@ -5,81 +5,87 @@ const initiaState = {
     error: null,
     loading: false,
     message: "",
-    averageRating: null
+    averageRating: null,
+    facebook: null,
+    email: null,
+    twitter: null
```

Comment:
I feel like all this information is already contained in the article variable here in this article reducer state so this is kind of some redundancy. Could I get a cleary reason though to why you decided to do this?

Reply:
working on this as well